### ISSUE TITLE
{Freeway-Bug}[Adaptive Cards- Import- Activity Update]: Luminosity ratio of selected date on Calendar Picker under 'Activity Update' pane is 1.2:1 which is less than 3:1.

### ISSUE THREAD

**Issue by Bhaskar301101** [microsoft/AdaptiveCards#8464]
### Target Platforms

Android

### SDK Version

2023.03.31.1

### Application Name

Adaptive Card

### Problem Description

Test Environment
Device: S21+ 5G
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.03.31.1

Repro Steps:
1. Launch the Adaptive cards app on Android.
2.  Navigate to 'Import' control and select it.
3. Navigate to 'Sample cards' and select it.
4. Navigate to available dropdown under 'Sample cards' and select it.
5. Navigate all the controls under dropdown and select 'Activity Update'.
6. Navigate to 'Visual' tab control and select it.
7. Navigate all the controls under 'Visual'.
8. Navigate to Date Picker and select a date.
9. Observe the issue for selected date issue with the tool.

Actual Result:
Luminosity ratio of selected date on Calendar Picker under 'Activity Update' pane is 1.2:1 which is less than 3:1.

Expected Result:
Luminosity ratio of selected date on Calendar Picker under 'Activity Update' pane should be greater than or equal to 3:1.

Note: Same issue is also repro for Selected Time also.

User Experience:
User with low vision might face difficulty to understand/ If the color contrast ratio is less than 3:1 for the focus indicator boundary with keyboard on selected date.

WCAG Reference:
https://www.w3.org/WAI/WCAG21/Understanding/non-text-contrast

Have feedback to share on Bugs? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-chisha%40microsoft.com%7c1334b110f98744a7f5df08d968eb842b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656183104980596%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=r4ndSHk/LrBDwzi7FO%2BjKN0DCnqUSUcELBvGRkgc5CU%3D&reserved=0)"

### Screenshots

![Luminosity ratio of keyboard focus boundary on selected date is less than 3 1](https://user-images.githubusercontent.com/89897257/232704345-8a383674-16f9-4580-93df-8ca6060330ae.png)

### Card JSON

```JSON
NA
```

**Comment by jwoo-msft** [comment]
it's native Android UI issue we can't fix.
